server/lib/budget.js

```diff
@@ -204,7 +226,7 @@ export function getSumCollectivesAmountSpent(
     : 'amountInHostCurrency';
   const transactionType = 'DEBIT_WITHOUT_HOST_FEE';
 
-  return sumCollectivesTransactions(collectiveIds, {
+  const results = await sumCollectivesTransactions(collectiveIds, {
```

Comment:
Can we run the fallback query only for the missing collective ids?
```suggestion
  const results = await sumCollectivesTransactions(missingCollectiveIds, {
```

Reply:
Good catch!